Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -203,19 +203,26 @@ darker input fields:           #505050
     .z-cse .x-panel-header-default {
         background-color: #f3f3f3;
     }
-/* <ZEN-29844> floating windows light theme inheritance */
+/* <ZEN-29844>,<ZEN-30467> floating windows light theme inheritance */
 .z-cse .x-window,
 .z-cse .x-window-header {
     background-image: none;
-    background: #dddddd;
```

Comment:
can these backgrounds be transparent?

Reply:
in case of transparent background - text is bad readable on window body.